src/include/btmem.h

```diff
@@ -1075,7 +1075,7 @@ struct __wt_update {
 
 /* AUTOMATIC FLAG VALUE GENERATION START */
 #define WT_UPDATE_HS 0x01u                       /* Update has been written to history store. */
-#define WT_UPDATE_MASKED_BY_NON_TS_UPDATE 0x02u  /* Update masked by updates without timestamp. */
+#define WT_UPDATE_OBSOLETE 0x02u                 /* Update that is obsolete. */
```

Comment:
The indenting here doesn't look right.

Reply:
I don't see any problem here. Can you elaborate?

Reply:
This is probably my past coming back to haunt me. clang should get it right.